### PR TITLE
Remove unnecessary new keywords

### PR DESCRIPTION
**Pull request by VEINHORN**

### CODE REVIEW

File: akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala [hunk 1]
```diff
@@ -120,7 +120,7 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
   /**
    * Is this node the leader?
    */
-  def isLeader: Boolean = leader == Some(selfAddress)
+  def isLeader: Boolean = leader.contains(selfAddress)
```
Comment (patriknw) [inline akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala]: +1 (it was written before Scala 2.11)

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
OK TO TEST

**Comment by patriknw** [comment]
@VEINHORN can't use that in `.java` :)

```
[error] /localhome/jenkinsakka/workspace/pr-validator-per-commit-jenkins/akka-docs/rst/java/code/docs/pattern/SupervisedAsk.java:85: cannot find symbol
[error]   symbol:   method ActorKilledException(java.lang.String)
[error]   location: class docs.pattern.SupervisedAsk.AskSupervisor
[error]         Throwable ex = ActorKilledException("Target actor terminated.");
```

**Comment by VEINHORN** [comment]
@patriknw fixed. :)

**Comment by patriknw** [comment]
PLS BUILD

**Pull request merged**